apps/mull-api/src/app/user/user.service.ts

```diff
@@ -12,11 +13,11 @@ export class UserService {
     private userRepository: Repository<User>
   ) {}
 
-  findAll(): Promise<User[]> {
+  users(): Promise<User[]> {
     return this.userRepository.find();
   }
 
-  findOne(id: number): Promise<User> {
+  user(id: number): Promise<User> {
```

Comment:
How do we feel about `findUser`? with just `user` I had to take a second to think and go see what the function did
```suggestion
  findUser(id: number): Promise<User> {
```

Reply:
If we change it here, let's change the below to `findFriends`

Reply:
sure I'm good with `findUser`

Reply:
> Overall @r-saba I don't mind your suggestions, but I find that they might be a bit too verbose. My main concern was having identical names between the services and resolvers, not really the specific names themselves

Ahh I see, my concern was more on the names. I noticed teammates getting confused while using the graphql editor on what exactly the query or mutation did.

Reply:
I'm thinking of putting a name like `getUser` for the service name